apps/client/build/run_action.mjs

```diff
@@ -31,7 +31,7 @@ const resolveActionPath = async actionPath => {
     return actionPath;
```

Comment:
The cross-app build stuff should go under infrastructure.

Reply:
In this case you are moving something from outside the app code into the app code. Since you are moving it already, move it to the right destination. Or leave it where it is for now if you prefer to move that later.

Reply:
I think we have alignment on the app code. No need to block the move of what's in `src/{cordova/electron/www}` to `apps/client`.

Reply:
The infrastructure could move one level up, but that could be a different PR. The build folder must be split. Some of that is app-specific, some can go into infrastructure. All of that can be incremental. I think moving everything in one move is going to take longer, versus moving each part independently